jcm/model.py

```diff
@@ -186,10 +186,36 @@ def conserve_global_mean_surface_pressure(u, u_next):
                 log_surface_pressure=u_next.log_surface_pressure.at[0, 0, 0].set(u.log_surface_pressure[0, 0, 0])
             )
         
+        # create diffusion filter function handles
+        diffuse_div = DiffusionFilter.make_diffusion_fn(
```

Comment:
I'm not sure about this pattern. I think partly because it's a class method and I don't think it should be, but also that it's having to unpack specific parts of the diffusionFilter object for each call. I guess part of the problem is that we can't have real methods on pytrees? Perhaps it can be a (private) method on Model?